app/views/user/mygardens/plants/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1> <%= @garden.name %> </h1>
+<% if @garden.plants.empty? %>
+	<h3> No plants in this garden.  Find something you would like to grow and add them to keep track of what you have planted </h3>
+<% else %>
+	<% @garden.plants.each do |plant| %>
```

Comment:
do we want to add the image here as well?

Reply:
probably a future iteration.